CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## Upcoming
```

Comment:
Changelog here is generated automatically during release, so this should be removed.